Lesson_5/src/com/cycles/exercise10.java

```diff
@@ -0,0 +1,31 @@
+package com.cycles;
+
+import com.sun.org.apache.xpath.internal.res.XPATHErrorResources_sv;
+
+public class exercise10 {
+
+    public static void main(String[] args) {
+
+
+        // Посчитать факториал числа в границах от 10 до 15
+
+        int a = 10;
+        int b = a;
+        int c= 15;
+        System.out.print("число произведения" +b);
+
+        while ( b<=c){
+            a = a*b;
+            b++;
+            System.out.println(" произведение = " + a);
```

Comment:
10! should be equal to  3628800
11! - 39916800
and ctr. you just multiplying numbers and I don't know how this operation should ba called :)